src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1555,37 +1569,48 @@ export class ResultsViewPageStore {
         invoke: () => fetchCnaOncoKbData(this.uniqueSampleKeyToTumorType.result!, this.oncoKbAnnotatedGenes.result!, this.discreteCNAData)
     }, ONCOKB_DEFAULT);
 
+    //we need seperate oncokb data because oncoprint requires onkb queries across cancertype
+    //mutations tab the opposite
+    readonly cnaOncoKbDataForOncoprint = remoteData<IOncoKbData>({
+        await: ()=> [
+            this.uniqueSampleKeyToTumorType,
+            this.oncoKbAnnotatedGenes,
+            this.discreteCNAData
+        ],
+        invoke: () => fetchCnaOncoKbData({}, this.oncoKbAnnotatedGenes.result!, this.discreteCNAData)
+    }, ONCOKB_DEFAULT);
+
     readonly getOncoKbMutationAnnotation = remoteData({
         await: ()=>[
-            this.oncoKbData
+            this.oncoKbDataForOncoprint
         ],
         invoke: ()=>{
             return Promise.resolve((mutation:Mutation)=>{
-                const uniqueSampleKeyToTumorType = this.oncoKbData.result.uniqueSampleKeyToTumorType!;
+                const uniqueSampleKeyToTumorType = this.oncoKbDataForOncoprint.result.uniqueSampleKeyToTumorType!;
                 const id = generateQueryVariantId(
                     mutation.entrezGeneId,
                     cancerTypeForOncoKb(mutation.uniqueSampleKey, uniqueSampleKeyToTumorType),
                     mutation.proteinChange,
                     mutation.mutationType
                 );
-                return this.oncoKbData.result!.indicatorMap![id];
+                return this.oncoKbDataForOncoprint.result!.indicatorMap![id];
             });
         }
     });
 
     readonly getOncoKbCnaAnnotation = remoteData({
```

Comment:
getOncoKbCnaAnnotationForOncoprint (maybe we should consider moving this stuff into the oncoprint..., maybe later)